### PR TITLE
Added lrphase package

### PR DESCRIPTION
**Pull request by adadiehl**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by adadiehl** [comment]
@BiocondaBot please add label

File: recipes/lrphase/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  commands:
+    - LRphase --help
+
+about:
+  home: "https://github.com/Boyle-Lab/LRphase.git"
+  license: MIT
+  license_family: MIT
+  license_file: 
+  summary: "Phasing individual long reads using known haplotype information."
+  doc_url:
```
Comment (dpryan79) [inline recipes/lrphase/meta.yaml:52]: Please delete from `doc_url:` to the end of the file.

Comment (adadiehl) [reply recipes/lrphase/meta.yaml]: Made the changes you asked for. All should be ready to go now.

File: recipes/lrphase/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 2375aeb05cac517adb6fe6e9b02574ea98efd298586dfd54a7929c1d8605c60f
+
+build:
+  number: 0
+  entry_points:
+    - LRphase = LRphase.cli:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: generic
```
Comment (dpryan79) [inline recipes/lrphase/meta.yaml:17]: I missed this before, it should be `noarch: python`

**Review by dpryan79 — CHANGES_REQUESTED** [review]
One last change

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**